runner/host/lsmod_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Please be sure to include the SPDX line commented above in lsmod.go here as well.